UKHomeOffice/sgmr-service
# SPCR-206 Change flow for cancelling a voyage

## Description
The journey for cancelling a voyage needs to be changed. After clicking on a voyage plan and clicking the cancel button, the user should be taken to a confirmation page and after choosing 'yes' to confirm, the user should be taken back to the voyage plans page and a success notification should appear. 

In some cases, incomplete plans and plans in the past cannot be cancelled. If a cancellation IS successful, the user is directed to a black page.

### To test
- Create a voyage plan but do not complete all the pages
- Go to voyage reports
- Click on the voyage
- On the check answers page, click on cancel voyage
- It should take you to a confirmation page
- Click yes and then continue
- It should take you to the voyage plan reports page with a success banner confirming cancellation
- Click on another tab and back to voyage plans
- The banner should no longer be visible. 
- The draft voyage should also move to cancelled
-
- Repeat but with a completed report
-
- On the confirmation page, if no is clicked
- You should be redirected to the voyage plan page and no banner should be visible

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
